src/App.tsx

```diff
@@ -1,32 +1,17 @@
-import * as React from 'react';
 import Container from '@material-ui/core/Container';
 import Typography from '@material-ui/core/Typography';
 import Box from '@material-ui/core/Box';
-import Link from '@material-ui/core/Link';
-import ProTip from './ProTip';
+import AddExerciseForm from './AddExerciseForm';
 
-function Copyright() {
-  return (
-    <Typography variant="body2" color="text.secondary" align="center">
-      {'Copyright © '}
-      <Link color="inherit" href="https://material-ui.com/">
-        Your Website
-      </Link>{' '}
-      {new Date().getFullYear()}.
-    </Typography>
-  );
-}
+const App = () => (
+  <Container maxWidth="sm">
```

Comment:
Not really sure what this box and container components are doing, are they necessary? Maybe the app can just be wrapped with a single Container component with no additional props.